vector/src/main/java/im/vector/app/features/spaces/create/CreateSpaceViewModel.kt

```diff
@@ -112,18 +137,24 @@ class CreateSpaceViewModel @AssistedInject constructor(
                     )
                 }
             }
-            is CreateSpaceAction.SetAvatar -> {
+            is CreateSpaceAction.SetAvatar              -> {
                 setState { copy(avatarUri = action.uri) }
             }
-            is CreateSpaceAction.SetSpaceTopology -> {
+            is CreateSpaceAction.SetSpaceTopology       -> {
                 handleSetTopology(action)
             }
         }.exhaustive
     }
 
+    private fun getAliasFromName(name: String): String {
+        return Regex("\\s").replace(name.lowercase(), "_").let {
+            "[^a-z0-9._%#@=+-]".toRegex().replace(it, "")
+        }
```

Comment:
Maybe move that to the Matrix SDK somewhere?

Reply:
moved to MatrixPatterns

Reply:
Thanks